docs/tutorialkit.dev/astro.config.ts

```diff
@@ -19,11 +19,7 @@ export default defineConfig({
         light: './src/assets/brand/tutorialkit-logo-light.svg',
         replacesTitle: true,
       },
-      plugins: [
-        starlightLinksValidator({
-          exclude: ['../../guides/**/*'],
-        }),
-      ],
+      plugins: [starlightLinksValidator()],
```

Comment:
By using absolute paths instead, this seems to work. I only just noticed that other pages also use absolute paths 🤷 .